programmers/2일차/수박수박수박수.js

```diff
@@ -0,0 +1,23 @@
+function solution (n) {
+    const word  = "수박";
+    const split_word = word.split("");
+    const answer =[];
+    let count = 0;
+    while(n>count) {
+        for (let i of split_word) {
```

Comment:
```suggestion
        for (let word of split_word) {
```
for of 를 사용하셨군요  for of를 사용할때는   일반적으로  for문 반복문에서 사용하는  i를  쓰는 것을 지양해요 
주로  for of는 배열의 값에 접근하여 순회하기때문에  주로  그 배열의 값들을  통칭하는 변수명을 사용합니다
같은 이유로 이 코드에서 순회하는 배열은  단어들이 들어 있는 배열이기에  저같은 경우는 word를 사용을 했습니다